README.md

```diff
@@ -41,31 +41,31 @@ see [OpenELIS-Docker setup](https://github.com/I-TECH-UW/openelis-docker)
 
     docker-compose -f build.docker-compose.yml up -d --build
 
-#### Running OpenELIS Global2 using docker compose With locally compiled/built Artifacts (ie the War file and React code)
+#### Running OpenELIS Global2 using docker compose with locally compiled/built Artifacts (ie the War file and React code)
 
 1.  Fork the
     [OpenELIS-Global Repository](https://github.com/I-TECH-UW/OpenELIS-Global-2.git)
     and clone the forked repo. The `username` below is the `username` of your
     Github profile.
 
-         git clone https://github.com/username/OpenELIS-Global-2.git
+        git clone https://github.com/username/OpenELIS-Global-2.git
 
-1.  innitialize and build sub modules
+2.  initialize and build sub modules
```

Comment:
Dont change the numbering of lists to a Munaual numbering.
Markdown will automatically render the list with correct numbering when displayed.
This helps for easy maintainace where you dont have to manually manage the numbering incase items change order or change number